packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -265,15 +265,16 @@ export default class SalesforceCRMService implements CRM {
         );
       }
     }
-    if (appOptions.onBookingWriteToRecord && appOptions.onBookingWriteToRecordFields) {
-      await this.writeToPersonRecord(
+    if (appOptions?.onBookingWriteToRecord && appOptions?.onBookingWriteToRecordFields) {
+      await this.writeToRecord({
```

Comment:
Refactored `this.writeToPersonRecord` to just `this.writeToRecord` so we can reuse the method to write to a variety of records.